### ISSUE TITLE
type string GVariant«(i,i)» invalid

### ISSUE THREAD

**Issue by simona70** [hongquan/CoBang#84]
/usr/share/glib-2.0/schemas/vn.hoabinh.quan.CoBang.gschema.xml:4:41  Error at line 4 char 41: String type GVariant«(i,i)» invalid

**Comment by LFd3v** [comment]
Hi, there. I do not know when this error started (I have installed CoBang recently), but the error is caused by this line in the code:

https://github.com/hongquan/CoBang/blob/da3daf90d61551d69e680e39784446b4ae2381e0/data/vn.hoabinh.quan.CoBang.gschema.xml#L4

This fixed the error for me (devs probably will want to change the _default_ values):

```
        <key name="window-size" type="(ii)">
            <default>(300,120)</default>
        </key>
```

**Comment by hongquan** [comment]
Thank you. I accidently included a half-baked code.

**Comment by hongquan** [comment]
Fixed by 9072b3ddfecbd88ec09d750ad736d457dc97a257

**Comment by hongquan** [comment]
Fix is released in v0.10.5.
